command/genesis/genesis.go

```diff
@@ -65,7 +65,8 @@ func setFlags(cmd *cobra.Command) {
 		stakeFlag,
 		[]string{},
 		fmt.Sprintf(
-			"the staked accounts and balances (format: <address>[:<stake>]). Default staked balance: %d",
+			"the staked accounts and balances (format: <address>[:<stake>]). "+
+				"Default staked balance if token is not mintable: %d",
```

Comment:
Not sure that I understand this description:
> Default staked balance if token is not mintable

Were you about to write that if token is locally mintable?
E.g.
```suggestion
				"Default staked balance if stake token is minted on the local chain: %d",
```

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/49/commits/4fbda926da625bc331636ac6f3c059cd75f1dd80